crates/bevy-inspector-egui/src/reflect_inspector/mod.rs

```diff
@@ -366,13 +366,6 @@ impl InspectorUi<'_, '_> {
             }
             TypeInfo::Enum(info) => self.ui_for_enum_many(info, ui, id, options, values, projector),
             TypeInfo::Value(info) => self.ui_for_value_many(info, ui, id, options),
-            TypeInfo::Dynamic(_) => {
```

Comment:
As I understand it, this should technically be replaced with an `is_dynamic` check, but I could not figure out whether this is actually relevant anymore.

Reply:
I am unsure. (inherited from https://github.com/mvlabat/bevy_egui/pull/188 )

Reply:
I'm not too familiar with `Reflect` specifics, thanks for pointing that out